sirepo/uri_router.py

```diff
@@ -116,6 +118,11 @@ def init(app, simulation_db):
 
     for n in _REQUIRED_MODULES + tuple(sorted(feature_config.cfg().api_modules)):
         register_api_module(importlib.import_module('sirepo.' + n))
+    for f in pkio.sorted_glob(simulation_db.STATIC_FOLDER.join('en','*.html')):
+        @sirepo.api_perm.allow_visitor
```

Comment:
I don't like this style, because it obscures the names. Right now we can find all apis with grep `^def api_`. It's very useful to be able to do this.